docs/es/docs/python-types.md

```diff
@@ -237,28 +272,49 @@ Una vez más tendrás todo el soporte del editor:
 
 ## Modelos de Pydantic
 
-<a href="https://pydantic-docs.helpmanual.io/" class="external-link" target="_blank">Pydantic</a> es una library de Python para llevar a cabo validación de datos.
+<a href="https://pydantic-docs.helpmanual.io/" class="external-link" target="_blank">Pydantic</a> es una libreria de Python utilizada para llevar a cabo validación de datos.
```

Comment:
Please revert the word "librería", even though it's sometimes used in Spanish, it's incorrect, it should be "biblioteca", but that's not commonly used, that's why I use the technical word in English.